src/languages/en/ui.js

```diff
@@ -32,7 +32,7 @@ export const LAST_NAME = 'Last Name';
 export const ROLE = 'Role';
 export const EMAIL = 'Email';
 export const JOB_TITLE = 'Job Title';
-export const WEEKLY_COMMITTED_HRS = 'Weekly Committed Hrs';
+export const WEEKLY_COMMITTED_HRS = 'Wkly Committed Hrs';
```

Comment:
Revert this change. you can see the left side and the right side of the equation are almost same, except for capitalization.